common/src/main/kotlin/okio/System.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (C) 2018 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package okio
+
+expect fun arraycopy(src: ByteArray, srcPos: Int, dest: ByteArray, destPos: Int, length: Int)
```

Comment:
internal ?

Reply:
Yeah, I wonder if it's gonna work as expected, since technically those are different modules

Reply:
Not a bug. You can't cross modules with `internal`.

Reply:
Makes sense, hence we should leave it public and ensure it's not exposed as a transitive dependency

Reply:
Why are we adding a transitive dependency instead of making the okio module multiplatform?

Reply:
Good point, I guess a better setup would be `okio-common` and `okio` itself being the JVM module

Reply:
I've been using nested projects for multiplatform if that interests you:
```
okio/
  js/
  jvm/
  src/
```